pkg/flavor/flavor.go

```diff
@@ -72,10 +72,26 @@ func (m *methods) FindProfile(vm *virtv1.VirtualMachine) (*flavorv1alpha1.Virtua
 	}
 }
 
-func (m *methods) ApplyToVmi(field *k8sfield.Path, profile *flavorv1alpha1.VirtualMachineFlavorProfile, vmiSpec *virtv1.VirtualMachineInstanceSpec) Conflicts {
+func (m *methods) ApplyToVmi(field *k8sfield.Path, profile *flavorv1alpha1.VirtualMachineFlavorProfile, vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) Conflicts {
 	var conflicts Conflicts
+	var flavor string
 
-	conflicts = append(conflicts, applyCpu(field, profile, vmiSpec)...)
+	if vm.Spec.Flavor != nil {
+		flavor = strings.ToLower(vm.Spec.Flavor.Kind)
```

Comment:
if `kind` is empty, this should default virtualmachineclusterflavor

```
        // Kind specifies which flavor resource is referenced.
        // Allowed values are: "VirtualMachineFlavor" and "VirtualMachineClusterFlavor".
        // If not specified, "VirtualMachineClusterFlavor" is used by default.
        //
        // +optional
        Kind string `json:"kind,omitempty"`

```

Reply:
I forgot to ask about the default before the holiday.   Will move the default to `VirtualMachineClusterFlavor`.